lib/client/connection.js

```diff
@@ -5,6 +5,8 @@ var emitter = require('../emitter');
 var ShareDBError = require('../error');
 var types = require('../types');
 var util = require('../util');
+// TODO: Move back to the 'sharedb' logger name in v1.0
```

Comment:
I don't get why the `'sharedb'` logger name can't be used now. Where is the conflict?

This should probably be tracked in an issue, or added to the 1.0 TODO list.

Related https://github.com/share/sharedb/issues/144

Reply:
The problem is that I want to remove the `debug` flag in this class, which is sort of already trying to provide some control over log levels. In fact, leaving in the `debug` flag creates a lot of confusion, because if you set your log level to `TRACE`, but forget to set `debug = true`, then you won't get any output.

However, removing `debug` is technically a breaking change, so for now we need to give `Connection` its own logger which is always at `DEBUG`, and who output is actually controlled by the `debug` flag.